src/components/screens/editor/editor-screen.tsx

```diff
@@ -1,12 +1,273 @@
-import { Typography } from "@mui/material";
+import { Accordion, AccordionDetails, AccordionSummary, Divider, Grid, List, ListItem, Typography } from "@mui/material";
+import { ListRounded, KeyboardArrowDown, Settings, DateRangeRounded, PersonRounded, Add } from "@mui/icons-material";
+import { useApiClient } from "app/hooks";
+import { ErrorContext } from "components/contexts/error-handler";
 import strings from "localization/strings";
-import React from "react";
+import React, { useContext, useEffect, useState } from "react";
+import Api from "api";
+import { NavigationTabContainer } from "styled/layouts/navigations";
+import NavigationTab from "components/layouts/navigations/navigation-tab";
+import { Metaform, MetaformVersion, MetaformVersionType } from "generated/client";
+import { AdminFormListStack, AdminFormTypographyField } from "styled/react-components/react-components";
+import { DataGrid, GridColDef } from "@mui/x-data-grid";
+import { NewUserButton } from "styled/layouts/admin-layout";
+import moment from "moment";
+import EditorScreenDrawer from "./editor-screen-drawer";
+import { useNavigate } from "react-router-dom";
+
+/**
+ * Interface for single Metaform row
+ */
+interface Row {
+  metaform: Metaform;
+  versions: MetaformVersion[];
+}
 
 /**
  * Editor screen component
  */
-const EditorScreen: React.FC = () => (
-  <Typography>{ strings.generic.notImplemented }</Typography>
-);
+const EditorScreen: React.FC = () => {
+  const errorContext = useContext(ErrorContext);
+  const navigate = useNavigate();
+
+  const apiClient = useApiClient(Api.getApiClient);
+  const { metaformsApi, versionsApi } = apiClient;
+
+  const [ metaforms, setMetaforms ] = useState<Row[]>([]);
```

Comment:
Maybe better describing would be metaformRows

Reply:
Done.